src/newrelic/processor_test.go

```diff
@@ -242,10 +243,19 @@ func TestProcessorHarvestCustomEvents(t *testing.T) {
 func TestProcessorHarvestLogEvents(t *testing.T) {
 	m := NewMockedProcessor(1)
 
-	m.DoAppInfo(t, nil, AppStateUnknown)
+	appInfo := sampleAppInfo
+	appInfo.AgentEventLimits = collector.EventConfigs{
+		LogEventConfig: collector.Event{
+			Limit:        1000,
+			ReportPeriod: 5000,
+		},
+	}
+
```

Comment:
Based on the duplication of this block in this file, it may be best to abstract it out and have it return an appInfo with Limit as an arg, something like the following:

```
func GetLogEventAppInfo(int limit) info *AppInfo {
	appInfo := sampleAppInfo
	appInfo.AgentEventLimits = collector.EventConfigs{
		LogEventConfig: collector.Event{
			Limit:        1000,
			ReportPeriod: 5000,
		},
	}

	return appInfo
```

Potentially could event make it a select case and handle multiple different event types.

Reply:
Thanks for the suggestion I have refactored the code and addressed this and the other examples of the AppInfo struct being initialized this way with a single helper function in commit [7bd8ba](https://github.com/newrelic/newrelic-php-agent/pull/497/commits/7bd8bad464752d428c6d8fa2960661d85c6fe3f5).

Reply:
👍